apartmenttherapy/order-service
# Elaine/prevent na n

#### What does this PR do & why?
Pulling the order's `offerAmount` from the correct place. Also adding just a little number validation.

#### How can this be manually tested?
- Tests should run.
- You should be able to accept an order for pickup and have its `totalAmount` set to the correct value in the db (vs `NaN`).

#### Any questions or concerns about this change?
*(Any significant changes you're introducing? What side effects might this have? What other parts of the app could this affect?)*


#### Deployment Notes
*(e.g. Does this require a migration or special infrastructure change? Clearing the cache? Third party service?)*


#### Link to story card and other documentation
https://app.asana.com/0/694906336753887/1114069197375199

#### GIF


#### Have you remembered to...?
